Analytical_Engine/Query/Graph/Relation.cs

```diff
@@ -46,6 +46,24 @@ public static partial class Query
         [Output("relations", "The collection of matching IRelations.")]
         public static List<IRelation> Relation(this Graph graph, IBHoMObject source, IBHoMObject target, RelationDirection relationDirection = RelationDirection.Forwards)
         {
+            if (graph == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the relations of a null graph.");
+                return null;
```

Comment:
Why not return an empty collection to follow the pattern?
```suggestion
                return new List<IRelation>();
```